src/main/java/com/google/devtools/build/lib/exec/AbstractSpawnStrategy.java

```diff
@@ -332,17 +330,7 @@ public void report(ProgressStatus state, String name) {
 
       // TODO(ulfjack): We should report more details to the UI.
```

Comment:
Do you think we can remove this TODO now? :)

Reply:
Yes!